modules/tiles/src/tileset/helpers/frame-state.ts

```diff
@@ -175,7 +175,7 @@ function closestPointOnPlane(
 
 function worldToCartesian(
   viewport: Viewport,
-  point: [number, number, number] | Vector3,
+  point: number[] | Vector3,
```

Comment:
Use https://github.com/visgl/loaders.gl/blob/f6811aebd390cc4326b8ea329f27d2a9e6e7eea6/modules/loader-utils/src/types.ts#L18 to make sure we also cover typed arrays? That type should be the same as the one in math.gl